ConsecutiveNumberSummer.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+
+public class ConsecutiveNumberSummer {
+    public static void findConsecutiveSums(int number) {

```

Comment:
define every method in brief through comments including short definition, input parameters and return values.